src/common/webrtc/api.rs

```diff
@@ -369,17 +425,21 @@ where
                 match candidate {
                     Ok(candidate) => {
                         if let Some(c) = candidate {
-                            log::debug!("received candidate : {}", &c);
                             tx.send(c).await.unwrap();
+                        } else {
+                            break;
                         }
                     }
+                    Err(WebRtcError::CannotParseCandidate) => continue,
```

Comment:
Is this case worth logging?

Reply:
actually it happens quite often as we reject mdns candidates,tcp based candidates and ipv6 candidates